MAUI/PDF-Viewer/Migration.md

```diff
@@ -318,25 +318,28 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
 </tr>
 <tr>
 <td>
-<div>{{'[PageChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_PageChanged)'| markdownify }},</div>
+<div>{{'[PageChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_PageChanged)'| markdownify }}</div>
 
 </td>
 <td>{{'[PropertyChanged](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.bindableobject.propertychanged)'| markdownify }}</td>
 <td> Occurs when a property in the PDF Viewer is modified. It allows you to detect page changes occurred in the PDF Viewer. The example below demonstrates how to implement this:
 
 {% highlight c# %}
 
-SfPdfViewer.PropertyChanged += PdfViewer_PropertyChanged;
+// Create an instance of the SfPdfViewer control
+SfPdfViewer pdfViewer = new SfPdfViewer();
 
-private void PdfViewer_PropertyChanged(object sender, PropertyChangedEventArgs e)
-{
-    // Check if the changed property is PageNumber
-    if (e.PropertyName == nameof(SfPdfViewer.PageNumber))
-    {
-        // Get the current page number
-        int currentPageNumber = SfPdfViewer.PageNumber;
+// Subscribe to the PropertyChanged event
+pdfViewer.PropertyChanged += PdfViewer_PropertyChanged;
 
-    }
+private void PdfViewer_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+{
+   // Check if the changed property is PageNumber
+   if (e.PropertyName == nameof(SfPdfViewer.PageNumber) && sender is SfPdfViewer)
+   {
+      // Get the current page number
+      int currentPageNumber = ((SfPdfViewer)sender).PageNumber;
```

Comment:
Don't use direct casting as it will cause exception. use **is** type of casting.

https://github.com/essential-studio/maui-pdfviewer/blob/development/maui/src/PDFViewer/Controls/SfPdfViewer.cs#L818

Reply:
addressed the feedback by changing the code to is type of casting